Dockerfile.fromswift53

```diff
@@ -0,0 +1,76 @@
+FROM swift:5.3-focal
```

Comment:
Were these new `Dockerfile`s used only for the investigation? I wonder if they can be removed now?

Reply:
Thank can be removed. You should see them gone in the next commit.